apps/condo/domains/organization/schema/OrganizationEmployeeRole.js

```diff
@@ -68,6 +68,8 @@ const OrganizationEmployeeRole = new GQLListSchema('OrganizationEmployeeRole', {
         canManageTicketComments: { type: Checkbox, defaultValue: true },
         canManageDivisions: { type: Checkbox, defaultValue: false },
         canShareTickets: { type: Checkbox, defaultValue: true },
+        canManageBillingReceipts: { type: Checkbox, defaultValue: false },
+        canManagePayments: { type: Checkbox, defaultValue: false },
```

Comment:
Only integration can manage aquiring data! we already have canManageIntegrations access

Reply:
Please refer to this comment: https://github.com/open-condo-software/condo/pull/688#discussion_r724340019